packages/fw-avatar/web-test-runner.config.js

```diff
@@ -0,0 +1,42 @@
+import { legacyPlugin } from "@web/dev-server-legacy";
+import { playwrightLauncher } from "@web/test-runner-playwright";
+
+const mode = process.env.MODE || "dev";
+if (!["dev", "prod"].includes(mode)) {
+  throw new Error(`MODE must be "dev" or "prod", was "${mode}"`);
+}
```

Comment:
Since we don't have different dev and prod env for this component, can remove this